.github/workflows/test_periodic.yml

```diff
@@ -14,6 +14,7 @@ env:
 
 jobs:
   build:
+    if: github.repository == 'ovn-org/ovn-kubernetes'
```

Comment:
`|| github.event_name == 'workflow_dispatch'`

Reply:
Which kind of custom event would you like to trigger there, antonio?

Reply:
He was pointing out that the job can be run in two ways: it's automatically run periodically, but you can also trigger it manually from the "Checks" page of the repo, and we want to allow that to work even in other repos. So the check needs to be "run if this is ovn-org/ovn-kubernetes, or if the run was manually requested"